grammars/julia.cson

```diff
@@ -130,7 +130,7 @@ repository:
             name: "entity.name.function.julia"
           "2":
             name: "support.type.julia"
-        match: "([[:alpha:]_][[:word:]\u207A-\u209C!\u2032]*)({(?:[^{}]|{(?:[^{}]|{[^{}]*})*})*})?\\((?=.*\\)(::[^\\s]+)?\\s*=(?![=>]))"
+        match: "([[:alpha:]_][[:word:]\u207A-\u209C!\u2032]*)({(?:[^{}]|{(?:[^{}]|{[^{}]*})*})*})?\\((?=.*\\)(::[^\\s]+)?(\\s*\\bwhere\\b\\s*[^\\s]+)?\\s*?=(?![=>]))"
```

Comment:
Sorry, I'm going to be picky one more time. This should really be `\\s+` as we require at least one space before the where:

```
julia> f3(A::Array{T}) where {T<:Any} = 3
f3 (generic function with 1 method)

julia> f2(A::Array{T})where {T<:Any} = 3
ERROR: syntax: space before "{" not allowed in "where {"
```